corehq/apps/data_interfaces/utils.py

```diff
@@ -34,3 +35,33 @@ def add_cases_to_case_group(domain, case_group_id, uploaded_data):
         case_group.save()
 
     return response
+
+
+def archive_forms(user, uploaded_data):
+    response = {
+        'errors': [],
+        'success': [],
+    }
+
+    for row in uploaded_data:
+        xform_id = row.get('form_id')
+        try:
+            xform = XFormInstance.get(xform_id)
```

Comment:
Also I notice that you don't have any domain checking. is that intentional? it seems like we should fail if a user tries to archive forms in a different domain from the URL they loaded the page from. obviously this will be super important the second the superuser restriction is lifted.